browsergym/webcanvas/src/browsergym/webcanvas/__init__.py

```diff
@@ -0,0 +1,16 @@
+from browsergym.core.registration import register_task
+
+# register the WebArena benchmark
+from . import config, task
+
+ALL_WEBCANVAS_TASK_IDS = []
+
+# register the WebArena benchmark
+for task_id in config.TASK_IDS:
+    gym_id = f"webcanvas.{task_id}"
```

Comment:
I'm a bit confused about naming. What is webcanvas? What is mind2web-live? My understanding is that webcanvas is a tool that allows for the creation of benchmarks. If so, the naming should be more explicit, like `webcanvas.mind2web-live.0`, which also allows for the addition of new webcanvas-based benchmarks in the future.